config/config_export_variables.go

```diff
@@ -62,20 +64,17 @@ func (conf *TerragruntConfig) ExportVariables(existingTerraformVariables map[str
 }
 
 func marshalTerraformVariables(existingTerraformVariables map[string]*configs.Variable, variables map[string]interface{}) ([]byte, error) {
-	lines := []string{}
+	file := hclwrite.NewEmptyFile()
 	for key, value := range variables {
 		if _, ok := existingTerraformVariables[key]; ok {
 			continue
 		}
-		lines = append(lines, fmt.Sprintf(`variable "%s" {`, key))
```

Comment:
Use hcl2's hclwrite package to write variables instead of appending strings manually